packages/mock-addon/.eslintignore

```diff
@@ -1,3 +1,6 @@
+# zx environment
+scripts/prepublish-checks.js
```

Comment:
Should not be linted, as it imports stuff like chalk that you don't have as a direct dependency and that's provided by zx